src/Emogrifier.php

```diff
@@ -179,7 +179,7 @@ class Emogrifier
         // classes, attributes, pseudo-classes (not pseudo-elements) except `:not`: worth 100
         '(?:\\.|\\[|(?<!:):(?!not\\())' => 100,
         // elements (not attribute values or `:not`), pseudo-elements: worth 1
-        '(?:(?<![="\':\\w-])|::)' => 1
+        '(?:(?<![="\':\\w\-])|::)' => 1
```

Comment:
Could you please use two backslashes for escaping the hyphen (also in the other places) so it's clear to the reader that this is escaping for the regexp, not for the PHP string?